File: programa.java
import java.util.Scanner;

public class Programa {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("¿Cuál es tu nombre? ");
        String nombre = sc.nextLine();

        System.out.print("¿En qué año naciste? ");
        int anio_nacimiento = sc.nextInt();

        int anio_actual = 2025;
        int edad = anio_actual - anio_nacimiento;

        System.out.println("Hola " + nombre + ", tienes " + edad + " años. Este programa está hecho en el lenguaje de programación: Java.");
    }
}

File: programa.c
#include <stdio.h>

int main() {
    char nombre[50];
    int anio_nacimiento, edad;
    int anio_actual = 2025;

    printf("¿Cuál es tu nombre? ");
    scanf("%s", nombre);

    printf("¿En qué año naciste? ");
    scanf("%d", &anio_nacimiento);

    edad = anio_actual - anio_nacimiento;

    printf("Hola %s, tienes %d años. Este programa está hecho en el lenguaje de programación: C.\n", nombre, edad);

    return 0;
}
